cloud/deploys/staging_azure/setup

```diff
@@ -12,7 +12,12 @@ source "${PROJECT_BASE_DIR}/cloud/azure/bin/lib.sh"
 readonly TF_VAR_FILE="${THIS_DIR}/staging.tfvars"
 readonly TEMPLATE_DIR="${PROJECT_BASE_DIR}/cloud/azure/templates/azure_saml_ses"
 readonly VARS_FILENAME="${THIS_DIR}/staging_azure_backend_vars"
-readonly RESOURCE_GROUP_NAME="tfstate"
+readonly TFSTATE_RESOURCE_GROUP_NAME="tfstate"
+
+readonly RESOURCE_GROUP_NAME="civiform-resourcegroup"
+readonly RESOURCE_GROUP_LOCATION="EastUS"
```

Comment:
question (blocking): is this location specified anywhere else or is it fully parameterized from here?

Reply:
I think these will eventually be read in from the config file that Susan is working on parsing, right? so setting these here is just a for-now thing. I forgot to leave a comment about that